src/components/Main/index.jsx

```diff
@@ -25,12 +24,15 @@ const selectorGenerator = new CssSelectorGenerator({
   selectors: ['id', 'tag', 'nthchild'],
 });
 
-const getWrappedText = (node) => {
-  const textNodes = filter(
-    node.childNodes, child => child.nodeType === child.TEXT_NODE);
-  const combined = trim(
-    textNodes.map(textNode => textNode.textContent).join(''));
-  return combined || null;
+const getWrappedText = (node, rich = false) => {
```

Comment:
i think that, given we have two uses for this function, i would make rich be an explicit argument rather than a default one.